src/Sonarr.Api.V3/Series/SeriesFolderAsRootFolderValidator.cs

```diff
@@ -11,19 +11,20 @@ public class SeriesFolderAsRootFolderValidator : PropertyValidator
         private readonly IBuildFileNames _fileNameBuilder;
 
         public SeriesFolderAsRootFolderValidator(IBuildFileNames fileNameBuilder)
-            : base("Root folder path contains series folder")
         {
             _fileNameBuilder = fileNameBuilder;
         }
 
+        protected override string GetDefaultMessageTemplate() => "Root folder path contains series folder";
+
         protected override bool IsValid(PropertyValidatorContext context)
         {
             if (context.PropertyValue == null)
             {
                 return true;
             }
 
-            var seriesResource = context.Instance as SeriesResource;
+            var seriesResource = context.InstanceToValidate as SeriesResource;
```

Comment:
From the documentation:
`Instance` was removed from `PropertyValidatorContext` - use `InstanceToValidate` instead